descriptors/ModuleDescriptor-template.json

```diff
@@ -93,8 +96,8 @@
             "invoice-storage.invoice-lines.item.post",
             "invoice-storage.invoice-lines.item.put",
             "invoice-storage.invoice-lines.collection.get",
-            "acquisitions-units-storage.units.collection.get",
-            "acquisitions-units-storage.memberships.collection.get"
+            "acquisitions-units.units.collection.get",
+            "acquisitions-units.memberships.collection.get"
```

Comment:
Unrelated to this PR, after you rebase with the master, there are few places where we are still referring `acquisitions-units-storage`. Could you please change that to `acquisitions-units`? [Line 27](https://github.com/folio-org/mod-invoice/blob/master/descriptors/ModuleDescriptor-template.json#L27), [Line 74](https://github.com/folio-org/mod-invoice/blob/master/descriptors/ModuleDescriptor-template.json#L74)

Reply:
Done